pilot/pkg/config/mcp/fakes/logger.go

```diff
@@ -0,0 +1,99 @@
+// Code generated by counterfeiter. DO NOT EDIT.
```

Comment:
If you think a fake logger is needed - please create a real fake, in the logger package we use (my understanding was that the logger package we use can be configured to not generate logs or whatever you need). 

Having a fake logger in each adapter is not practical and we want consistency.

Reply:
👍